### DISCUSSION TITLE
✨ SigNoz

### DISCUSSION THREAD

**Discussion by jasonpearce** [runtipi/runtipi-appstore#4487] [Requests]
### Description

SigNoz is an open-source observability platform that helps developers monitor applications and troubleshoot problems in their deployed applications. It provides a unified interface for metrics, traces, and logs, making it a powerful alternative to proprietary solutions like Datadog and New Relic. SigNoz is built on OpenTelemetry and uses ClickHouse for storage, ensuring high performance and scalability.

Features:
- Application Performance Monitoring (APM)
- Distributed Tracing
- Log Management
- Infrastructure Monitoring
- Exception Monitoring
- Alerts Management

### Official Website

https://signoz.io/

### Documentation link

https://signoz.io/docs/introduction/

### Last application release & date

v0.52.0 (2024-08-16)

### Application license

Apache License 2.0

### Source code repository link

https://github.com/SigNoz/signoz

### Docker image link

https://hub.docker.com/u/signoz

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Why Add SigNoz to RunTipi Appstore: 
Adding SigNoz to the RunTipi appstore will provide users with a robust, open-source solution for observability. Its comprehensive feature set and flexible deployment options make it an excellent choice for teams looking to enhance their monitoring and troubleshooting capabilities.

**Comment by JigSawFr** [comment]
@meienberger Something interesting
